ps_customtext.php

```diff
@@ -205,14 +205,25 @@ public function processSaveCustomText()
 
         foreach ($languages as $lang) {
             $text[$lang['id_lang']] = (string) Tools::getValue('text_' . $lang['id_lang']);
+
+            if (!Configuration::get('PS_ALLOW_HTML_IFRAME') && preg_match('/<iframe.*src=\"(.*)\".*><\/iframe>/isU', $text[$lang['id_lang']])) {
```

Comment:
Why using subpattern here `src=\"(.*)\"` ?
:thinking: I thought this expression should be used to extract a specific part of regex subject.